### PR TITLE
Update assign-policies.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/assign-policies.md [hunk 1]
```diff
@@ -135,8 +135,12 @@ Each policy type has its own set of cmdlets for managing it. Use the ```Grant-``
 
  Download and install the [Skype for Business Online PowerShell module](https://www.microsoft.com/en-us/download/details.aspx?id=39366) (if you haven't already), and then run the following to connect to Skype for Business Online and start a session.
 
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/assign-policies.md:139]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**